API/NOTES.md

```diff
@@ -1,4 +1,4 @@
-Shouldn't the API be compartmentalized component by component? Made less monolithic?
+Shouldn't the API be compartmentalized [component](https://github.com/coq/coq/blob/trunk/META.coq) by [component](https://github.com/coq/coq/blob/trunk/META.coq)?
```

Comment:
This is a completely trivial remark but why don't you do

    [component by component](https://github.com/coq/coq/blob/trunk/META.coq)

instead. With what you did instead, people will see two links and interested people may very well click on both, only to discover that it was twice the same link!